package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "document-viewer-ts",
-  "version": "0.3.1-legacy",
+  "version": "0.3.2-legacy",
```

Comment:
Do we want to switch back to the non legacy version now that we aren't supporting the older browsers?

Reply:
I think it might be better to continue to use the legacy version because we still import the code in older browsers, which I think could cause errors. I can potentially look into using babel like we do for other packages but the legacy version seems to be stable so I don't see a problem with using it.

Reply:
Nope thats a good point.